autockt/envs/ngspice_ledro_d_fc.py

```
@@ -0,0 +1,365 @@
+import random
+import psutil
+
+from multiprocessing.dummy import Pool as ThreadPool
+from collections import OrderedDict
+import yaml
+import yaml.constructor
+import statistics
+import os
+import IPython
+import itertools
+from eval_engines.util.core import *
```

Ruff (0.12.2)

`from eval_engines.util.core import *` used; unable to detect undefined names

(F403)